src/main/resources/project/wsadmin_scripts/deploy_enterprise_application.py

```diff
@@ -1,31 +1,31 @@
 import time
 
-appName='$[appName]'.replace(' ', '_')
-commands='$[commands]'
-additionalCommands='$[additionalcommands]'
-cluster='$[cluster]'
-serverList='$[serverList]'
-appPath='$[apppath]'
-precompileJSP='$[precompileJSP]'
-installDir='$[installDir]'
-distributeApp='$[distributeApp]'
-binaryConfig='$[binaryConfig]'
-deployBeans='$[deployBeans]'
-createMBeans='$[createMBeans]'
-overrideClassReloading='$[overrideClassReloading]'
-reloadInterval='$[reloadInterval]'
-deployWS='$[deployWS]'
-validateRefs='$[validateRefs]'
-processEmbConfig='$[processEmbConfig]'
-filePermissions='$[filePermissions]'
-customFilePermissions='$[customFilePermissions]'
-blaName='$[blaName]'
-autoResolveEJBRef='$[autoResolveEJBRef]'
-deployClientMod='$[deployClientMod]'
-clientDeployMode='$[clientDeployMode]'
-validateSchema='$[validateSchema]'
-mapModulesToServers='$[MapModulesToServers]'
-additionalDeployParams='$[additionalDeployParams]'
+appName=r'$[appName]'.replace(' ', '_')
+commands=r'$[commands]'
+additionalCommands=r'$[additionalcommands]'
+cluster=r'$[cluster]'
+serverList=r'$[serverList]'
+appPath=r'$[apppath]'
+precompileJSP=r'$[precompileJSP]'
+installDir=r'$[installDir]'
```

Comment:
Merging this to allow QA continue testing. Working on solution for trailing backslash